jetson/nav/rover.cpp

```diff
@@ -70,6 +82,11 @@ unsigned Rover::RoverStatus::getPathTargets()
   return mPathTargets;
 } // getPathTargets()
 
+int& Rover::RoverStatus::getMisses()
```

Comment:
This was the design pattern Andrew had done for past implementations, since we want to return the current count of misses, but also be able to modify it without having 2 separate functions, but rather 1 that essentially does both,

Reply:
eh